### DISCUSSION TITLE
Material-UI Failing in Parcel 2.0.0-rc.0

### DISCUSSION THREAD

**Discussion by TheNathanBlake** [parcel-bundler/parcel#6966] [Q&A]
Hi there,

I'm currently trying to build a project using Material-UI and Parcel V2 and PUGs (2.0.0-rc.0), and on running `parcel serve` I can successfully build the project, but trying to navigate to the locally deploy web page gives me the following error:

<img width="547" alt="Screen Shot 2021-09-20 at 6 41 43 PM" src="https://user-images.githubusercontent.com/9566223/134091802-8c918aa6-28e6-44a4-83a6-aa085e8e2eb3.png">

This seems to come from an unrelated colors import/export in the index.d.ts:
<img width="448" alt="Screen Shot 2021-09-20 at 6 43 08 PM" src="https://user-images.githubusercontent.com/9566223/134091849-ec8abb38-ead7-4cd5-af11-6a996303943e.png">
 
Normally I'd post this on the material-ui GitHub, but for some reason, reverting Parcel to v2.0.0-beta.3.1 solves this issue.

Does anybody know enough about Material UI or building with Parcel for React projects to point me in the right direction on this issue?

Here's a link to an example project that breaks: https://github.com/TheNathanBlake/parcel-pug-material-ui-example

**Comment by mischnic** [comment]
> I can also provide a dumbed-down code setup that fails the same way:

Please do. Also that package.json isn't really self-contained because of that `file:` reference

**Reply by TheNathanBlake** [reply]
True, I forgot about that.  Give me a minute, I'll set everything up and post it in a couple hours or so.

**Reply by TheNathanBlake** [reply]
Ok, updated the question post with a link to a breaking project.
